types/accesscontrol/resource.go

```diff
@@ -1,150 +1,189 @@
-	ResourceType_KV_ORACLE_PRICE_SNAPSHOT:             	{ResourceType_KV_ORACLE, []ResourceType{}},
-	ResourceType_KV_STAKING_VALIDATION_POWER:          	{ResourceType_KV_STAKING, []ResourceType{}},
-	ResourceType_KV_STAKING_TOTAL_POWER:               	{ResourceType_KV_STAKING, []ResourceType{}},
-	ResourceType_KV_STAKING_VALIDATORS_CON_ADDR:       	{ResourceType_KV_STAKING, []ResourceType{}},
-	ResourceType_KV_STAKING_UNBONDING_DELEGATION:     	{ResourceType_KV_STAKING, []ResourceType{}},
-	ResourceType_KV_STAKING_UNBONDING_DELEGATION_VAL: 	{ResourceType_KV_STAKING, []ResourceType{}},
-	ResourceType_KV_STAKING_REDELEGATION:              	{ResourceType_KV_STAKING, []ResourceType{}},
-	ResourceType_KV_STAKING_REDELEGATION_VAL_SRC:      	{ResourceType_KV_STAKING, []ResourceType{}},
-	ResourceType_KV_STAKING_REDELEGATION_VAL_DST:      	{ResourceType_KV_STAKING, []ResourceType{}},
-	ResourceType_KV_STAKING_REDELEGATION_QUEUE:        	{ResourceType_KV_STAKING, []ResourceType{}},
-	ResourceType_KV_STAKING_VALIDATOR_QUEUE:           	{ResourceType_KV_STAKING, []ResourceType{}},
-	ResourceType_KV_STAKING_HISTORICAL_INFO:           	{ResourceType_KV_STAKING, []ResourceType{}},
-	ResourceType_KV_STAKING_UNBONDING:                	{ResourceType_KV_STAKING, []ResourceType{}},
-	ResourceType_KV_DISTRIBUTION:                         {ResourceType_KV, []ResourceType{
-		ResourceType_KV_DISTRIBUTION_FEE_POOL,
-		ResourceType_KV_DISTRIBUTION_PROPOSER_KEY,
-		ResourceType_KV_DISTRIBUTION_OUTSTANDING_REWARDS,
-		ResourceType_KV_DISTRIBUTION_DELEGATOR_WITHDRAW_ADDR,
-		ResourceType_KV_DISTRIBUTION_DELEGATOR_STARTING_INFO,
-		ResourceType_KV_DISTRIBUTION_VAL_HISTORICAL_REWARDS,
-		ResourceType_KV_DISTRIBUTION_VAL_CURRENT_REWARDS,
-		ResourceType_KV_DISTRIBUTION_VAL_ACCUM_COMMISSION,
-		ResourceType_KV_DISTRIBUTION_SLASH_EVENT,
-	}},
-	ResourceType_KV_DISTRIBUTION_FEE_POOL:                {ResourceType_KV_DISTRIBUTION, []ResourceType{}},
-	ResourceType_KV_DISTRIBUTION_PROPOSER_KEY:            {ResourceType_KV_DISTRIBUTION, []ResourceType{}},
-	ResourceType_KV_DISTRIBUTION_OUTSTANDING_REWARDS:     {ResourceType_KV_DISTRIBUTION, []ResourceType{}},
-	ResourceType_KV_DISTRIBUTION_DELEGATOR_WITHDRAW_ADDR: {ResourceType_KV_DISTRIBUTION, []ResourceType{}},
-	ResourceType_KV_DISTRIBUTION_DELEGATOR_STARTING_INFO: {ResourceType_KV_DISTRIBUTION, []ResourceType{}},
-	ResourceType_KV_DISTRIBUTION_VAL_HISTORICAL_REWARDS:  {ResourceType_KV_DISTRIBUTION, []ResourceType{}},
-	ResourceType_KV_DISTRIBUTION_VAL_CURRENT_REWARDS:     {ResourceType_KV_DISTRIBUTION, []ResourceType{}},
-	ResourceType_KV_DISTRIBUTION_VAL_ACCUM_COMMISSION:    {ResourceType_KV_DISTRIBUTION, []ResourceType{}},
-	ResourceType_KV_DISTRIBUTION_SLASH_EVENT:             {ResourceType_KV_DISTRIBUTION, []ResourceType{}},
+    ResourceType_ANY: {ResourceType_ANY, []ResourceType{ResourceType_KV, ResourceType_Mem}},
+    ResourceType_KV: {ResourceType_ANY, []ResourceType{
```

Comment:
Converting indents to spaces to fix the formatting on git or else it looks weird